compiler-core/src/metadata/tests.rs

```diff
@@ -324,6 +329,7 @@ fn module_with_type_links() {
                     module: "a".into(),
                     parameters: vec![],
                     deprecation: Deprecation::NotDeprecated,
+                    documentation: Some("documentation".into()),
```

Comment:
Could we have a dedicated test for the documentation and also have some tests for the origin too please. Thanks